src/routes/overviews/bufnmf.svx

```diff
@@ -0,0 +1,134 @@
+---
+title: Audio Decomposition using BufNMF
+blurb: |
+    Decompose an audio buffer into component parts using non-negative matrix factorization.
+tags:
+    - decomposition
+flair: article
+---
+
+<script>
+    import Image from '$lib/components/Image.svelte';
+    import Audio from '$lib/components/Audio.svelte';
+</script>
+
+# Decomposition
```

Comment:
Because the title is already rendered in the layout (Audio Decomposition using BufNMF) it is necessary to make your first tag (Decomposition) here a `<h2>` or `##`. This also improves accessibility.

<img width="689" alt="image" src="https://user-images.githubusercontent.com/13281862/150223498-aa56e26f-e192-4286-a6b1-ad4abc49d699.png">